examples/zkdetect/main.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
Although, we could use build tags for such. Like `+integration`. But never mind, that can be done in the future.